src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: $card-width;
+  padding: 32px 16px 16px;
+  background-color: $color-white-accent;
+  border: 1px solid $color-white-anti-flash;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__image {
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: $color-main-accent;
+    font-size: $font-main-size;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 0.3s;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    color: $color-secondary;
+    font-size: $font-secondary-size;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &-reviews {
+      color: $color-main-accent;
+      font-size: $font-secondary-size;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-text {
+      margin-top: 24px;
+      color: $color-secondary;
+      font-size: $font-main-size;
+      line-height: 18px;
+    }
+
+    &-amount {
+      color: $color-main-accent;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin-top: 16px;
+    color: $color-white-accent;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 38px;
+    text-decoration: none;
+    text-transform: uppercase;
+    background-color: $color-blue-accent;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    transition: color 0.3s, background-color 0.3s, border-color 0.3s;
+
+    &:hover {
+      color: $color-blue-accent;
+      background-color: $color-white-accent;
+      border-color: $color-blue-accent;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
```

Comment:
Put `:hover` styles related to `.card` immediately after it.

Reply:
Fixed! 👍